frontend/public/components/RBAC/role.jsx

```diff
@@ -138,16 +138,14 @@ const BindingsListComponent = props => <BindingsList {...props} Header={BindingH
 
 export const BindingsForRolePage = (props) => {
   const {match: {params: {name, ns}}, kind} = props;
+  const resources = ns ? [{kind: 'RoleBinding', namespaced: true}] : [{kind: 'ClusterRoleBinding', namespaced: false}];
```

Comment:
I think you need to load both role binding and cluster role bindings when looking at a cluster role.